getting-started/running-gatewayd.md

```diff
@@ -120,7 +130,6 @@ Running GatewayD will produce the following log output, which means that Gateway
 GatewayD is running on the host, while PostgreSQL is running inside a container. You can run the following command to test it. Notice that `${DOCKER_HOST}` holds the IP address of the host machine, that is accessible from inside the container.
 
 ```bash
-DOCKER_HOST=$(ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/16//g')
```

Comment:
@sinadarbouy How about using this one-liner instead? It checks whether the `docker0` interface exists and acts accordingly. Then we can also remove the above explanations.
```bash
DOCKER_HOST=$(if ip addr show docker0 > /dev/null 2>&1; then ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/.*//'; else echo "host.docker.internal"; fi);
```